src/main/java/org/threadly/litesockets/udp/UDPClient.java

```diff
@@ -98,27 +156,157 @@ public void writeBlocking(ByteBuffer bb) {
   
   @Override
   public void setReader(Reader reader) {
-    super.setReader(reader);
+    this.reader = reader;
   }
   
   @Override
   public Reader getReader() {
-    return super.getReader();
+    return reader;
   }
   
   @Override
   public void setCloser(Closer closer) {
-    super.setCloser(closer);
+    this.closer = closer;
   }
   
   @Override
   public Closer getCloser() {
-    return super.getCloser();
+    return closer;
   }
 
   @Override
   public WireProtocol getProtocol() {
     return WireProtocol.UDP;
   }
+  
+  private static class ClientByteStats extends SimpleByteStats {
+    public ClientByteStats() {
+      super();
+    }
+
+    @Override
+    protected void addWrite(int size) {
+      if(size < 0) {
```

Comment:
If you want to save a little code, you can use "ArgumentVerifier" in org.threadly.utils.  It was designed to be a way to save jar space by reducing duplicated code.